package.json

```diff
@@ -25,6 +25,7 @@
     "grunt-jsonlint": "^1.0.4",
     "grunt-jsxhint": "^0.6.0",
     "grunt-lintspaces": "^0.7.0",
+    "node-lintspaces": "shaoshing/node-lintspaces",
```

Comment:
Well, I have no idea why this works (@mcilroy suggests `grunt` magic?) but it does seem to work. Note that I had to `rm -rf node_modules` and then `npm install` again for the change to take effect. But, hey, better linting!